app/pages/reservationSearch/day-reservations/reservation-row/_reservation-row.scss

```diff
@@ -44,6 +44,15 @@
     font-weight: regular;
   }
 
+  .icons {
+    & > * {
+      margin-right: 5px;
```

Comment:
- [ ] I think it could be clearer that the number is related to the user icon, and is not some separate thing. So, changing this to for example 10 px might help. Also, one possibility is to reduce the space between the user icon and the number. But you can test what looks good and decide.